pgql-spoofax/trans/normalize.str

```diff
@@ -439,30 +439,31 @@ rules
       commonPathExpressions' := <map(norm-common-path-expression(|variable-counter)); !CommonPathExpressions(<id>)> commonPathExpressions;
 
       expAsGroupVars := <?Some(GroupByClause(<id>)) <+ ![]> groupByClause;
-      expAsGroupVars' := <map(norm-ExpAsVar)> expAsGroupVars;
+      expAsGroupVars' := <map(norm-ExpAsVar(|version))> expAsGroupVars;
```

Comment:
What is version?

Reply:
it's the PGQL version (1.0, 1.1 or 1.2). For each version there are some different rules.
1.2 is just experimental and may not get released for a while..